### PR TITLE
Update localized-intellisense.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/core/install/localized-intellisense.md
Comment (Youssef1313) [inline docs/core/install/localized-intellisense.md]: I think these are already specified in *docfx.json*.

image

image

File: docs/core/install/localized-intellisense.md
Comment (Youssef1313) [inline docs/core/install/localized-intellisense.md]: Better to use it capitalized, as it has an acrynom ?
Feel free if you want to revert this change.

File: docs/core/install/localized-intellisense.md
Comment (Youssef1313) [reply docs/core/install/localized-intellisense.md]: By the way, In the CODEOWNERS files, core/install/** is set for @Thraka. But in the docfx, it's set for @mairaw as it's a sub-directory of core/**.

I 'm not sure if this was intended

File: docs/core/install/localized-intellisense.md
Comment (Youssef1313) [inline docs/core/install/localized-intellisense.md]: The steps below don't explain how to navigate to the .NET Core installation folder. They explain how to navigate to the moniker folder. The first step in navigating to the moniker folder is to navigate to the .NET Core installation folder.

File: docs/core/install/localized-intellisense.md
Comment (Youssef1313) [inline docs/core/install/localized-intellisense.md]: These are ordered steps. So, I changed it from unordered list to ordered list.

File: docs/core/install/localized-intellisense.md
Comment (Youssef1313) [inline docs/core/install/localized-intellisense.md]: nit: Just to be consistent with another sentence in the same article:

> This procedure requires that you have **administrator permission** to copy the IntelliSense files to the .NET Core installation folder.

File: docs/core/install/localized-intellisense.md
Comment (Youssef1313) [inline docs/core/install/localized-intellisense.md]: I think the correct name is "menu bar" not "main menu".

File: docs/core/install/localized-intellisense.md
Comment (Youssef1313) [inline docs/core/install/localized-intellisense.md]: **Environment** is not a folder.

File: docs/core/install/localized-intellisense.md
Comment (mairaw) [reply docs/core/install/localized-intellisense.md]: Yes it's by design. We can override folder owners this way.

File: docs/core/install/localized-intellisense.md
Comment (mairaw) [reply docs/core/install/localized-intellisense.md]: Our internal style guide says "IDE - Acronym for integrated development environment" so I think it was correct before

File: docs/core/install/localized-intellisense.md
Comment (mairaw) [reply docs/core/install/localized-intellisense.md]: Not sure if I follow.

File: docs/core/install/localized-intellisense.md
Comment (mairaw) [reply docs/core/install/localized-intellisense.md]: so can you use a similar sentence as used in the other sentence in this article?

File: docs/core/install/localized-intellisense.md
Comment (mairaw) [reply docs/core/install/localized-intellisense.md]: @gewarren do you know? I think I saw this in the VS docs if I'm not mistaken.

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @Youssef1313. I've left some comments for you to address before we can merge this.

File: docs/core/install/localized-intellisense.md
Comment (Youssef1313) [reply docs/core/install/localized-intellisense.md]: @mairaw, Okay I'll revert that one.

File: docs/core/install/localized-intellisense.md
Comment (Youssef1313) [reply docs/core/install/localized-intellisense.md]: This is what was before:

image

For me, I'll interpret it as if these red colored steps explain how to navigate to %ProgramFiles%\dotnet\packs, but they navigate to something like %ProgramFiles%\dotnet\packs\Microsoft.NETCore.App.Ref\3.1.0\ref\netcoreapp3.1

File: docs/core/install/localized-intellisense.md
Comment (gewarren) [reply docs/core/install/localized-intellisense.md]: Menu bar sounds more correct to me. It's not specific to VS though.

File: docs/core/install/localized-intellisense.md
Comment (Thraka) [reply docs/core/install/localized-intellisense.md]: I feel like this would be better if there was a path template up front: `%ProgramFiles%\dotnet\packs\[sdk-path]\[sdk-version]\ref\[sdk-moniker]`. Then define each parameter and give an example path. 

Also, we need to get this update for VS for Mac.

File: docs/core/install/localized-intellisense.md
Comment (Thraka) [reply docs/core/install/localized-intellisense.md]: @mairaw Should I take these over since they're in the install folder? I don't mind owning everything here.

**Review by Thraka — APPROVED** [review]
Besides what Maira pointed out, and my suggestion (which she can accept or decline) for the paths, I think this is good.

File: docs/core/install/localized-intellisense.md
Comment (mairaw) [reply docs/core/install/localized-intellisense.md]: Sounds good then. Let's keep the change.

File: docs/core/install/localized-intellisense.md
Comment (mairaw) [reply docs/core/install/localized-intellisense.md]: That's exactly where I wanted you to land:
%ProgramFiles%\dotnet\packs\Microsoft.NETCore.App.Ref\3.1.0\ref\netcoreapp3.1

So the original steps are correct.

Comment (mairaw) [reply docs/core/install/localized-intellisense.md]: I see where the confusion is. I think the only change I'd make is perhaps change line 31 to say something like

1. Navigate to the .NET Core Intellisense folder.

And then add a new line for opening the program files location before "Choose which SDK you want to install the IntelliSense for..."

- Navigate to the .NET Core installation folder. By default, it's under *%ProgramFiles%\dotnet\packs*.

Comment (mairaw) [inline docs/core/install/localized-intellisense.md]:
```suggestion
1. Navigate to the .NET Core Intellisense folder.
```

File: docs/core/install/localized-intellisense.md
Comment (mairaw) [reply docs/core/install/localized-intellisense.md]:
```suggestion
> To install, update, or modify Visual Studio, you must log on with an account that has administrator permission. For more information, see [User permissions and Visual Studio](/visualstudio/ide/user-permissions-and-visual-studio).
```

File: docs/core/install/localized-intellisense.md
Comment (mairaw) [inline docs/core/install/localized-intellisense.md]:
```suggestion
ms.date: 01/23/2020
```

**Comment by mairaw** [comment]
Left a few additional comments and I think this will be good to go!

**Review by mairaw — APPROVED** [review]
Thank you @Youssef1313!

**Pull request merged**
